samples/basic/expectReplies.ts

```diff
@@ -0,0 +1,52 @@
+import { startServer } from '@microsoft/agents-hosting-express'
+import { AgentApplication, CloudAdapter, loadAuthConfigFromEnv, MemoryStorage, TurnContext, TurnState } from '@microsoft/agents-hosting'
+import { DeliveryModes } from '@microsoft/agents-activity'
+
+const adapter = new CloudAdapter(loadAuthConfigFromEnv())
+// Patch process to log the invokeResponse sent to the client
+const originalProcess = adapter.process.bind(adapter)
+adapter.process = async (req, res, logic) => {
```

Comment:
not sure I follow.. Why we need to override process?

Reply:
We added a binding for the process to be able to log the invoke response, evidencing that the activities are included in it. This is the only method we found as the response is returned after all handlers are executed.

Reply:
we can log from the adapter.. no? 

One way to test this is to sending a request and assert the response